File: python/quadrante.py
print("Digite os valores das coordenadas X e Y:")
x = int(input())
y = int(input())

while x != 0 and y != 0:
    if y > 0 and x > 0:
        print("QUADRANTE Q1")
    elif y > 0 and x < 0:
        print("QUADRANTE Q2")
    elif y < 0 and x < 0:
        print("QUADRANTE Q3")
    elif y < 0 and x > 0:
        print("QUADRANTE Q4")

    print("Digite os valores das coordenadas X e Y:")
    x = int(input())
    y = int(input())

File: javascript/quadrante.js
let x, y;

do {
    x = parseInt(prompt("Coordenadas X"));
    y = parseInt(prompt("Coordenadas Y"));

    if (x > 0 && y > 0) {
        console.log("QUADRANTE Q1");
    } else if (x < 0 && y > 0) {
        console.log("QUADRANTE Q2");
    } else if (x < 0 && y < 0) {
        console.log("QUADRANTE Q3");
    } else if (x > 0 && y < 0) {
        console.log("QUADRANTE Q4");
    }
} while (x != 0 && y != 0);